doc/Learn/Tutorials/Http/HowTo-Refit.md

```diff
@@ -0,0 +1,219 @@
+
+## Step-by-steps
+
+> [!IMPORTANT]
+> This guide assumes you used the template wizard or `dotnet new unoapp` to create your solution. If not, it is recommended that you follow the [instructions](xref:Overview.Extensions) for creating an application from the template.
+
+### 1. Enable HTTP
+
+* When working with a complex application, centralized registration of your API endpoints is a good practice. This allows you to easily change the endpoint for a given service, and to easily add new services.
+
+* The first step to centrally registering any API endpoint is to enable HTTP on the `IHostBuilder`:
+    ```csharp
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var appBuilder = this.CreateBuilder(args)
+            .Configure(hostBuilder =>
+            {
+                hostBuilder.UseHttp();
+            });
+    ...
+    ```
+
+* This feature requires the `Uno.Extensions.Http` package. It uses [Microsoft Extensions](https://www.nuget.org/packages/Microsoft.Extensions.Http) for any HTTP-related [work](https://learn.microsoft.com/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests#benefits-of-using-ihttpclientfactory) such as naming or configuring the `HttpClient` instance associated with your endpoints.
+
+### 2. Define the live interface
+
+* Unlike standard HTTP endpoints, Refit endpoints you define will be registered as a service with the `AddRefitClient<T>()` extension method, where the type argument `T` corresponds to an interface you define.
+
+* This interface will be used to generate the strongly-typed client. It will be used to make requests to the web service and deserialize responses.
+
+* For the purposes of this tutorial, we will create and register an interface for the Chuck Norris facts web API 
+
+* Start by defining an interface `IChuckNorrisEndpoint` with a method `Search()` that returns a `Task` of type `ApiResponse<ChuckNorrisData>`:
+    ```csharp
+    [Headers("Content-Type: application/json")]
+    public interface IChuckNorrisEndpoint
+    {
+        [Get("/jokes/search")]
+        Task<ApiResponse<ChuckNorrisData>> Search(CancellationToken ct, [AliasAs("query")] string searchTerm);
+    }
+    ```
+
+    > [!NOTE]
+    > A class named `ChuckNorrisData` will be defined in the subsequent sections of this guide. It will be used to deserialize the response from the API. 
```

Comment:
I think the definition of ChuckNorrisData and other models should be done up front as part of creating this interface, and then move on to actually registering the interface with the host builder